src/DDTrace/Integrations/Curl/CurlIntegration.php

```diff
@@ -64,7 +64,7 @@ public function init()
                 }
 
                 $info = \curl_getinfo($ch);
-                $sanitizedUrl = Urls::sanitize($info['url']);
+                $sanitizedUrl = \DDtrace\Private_\util_url_sanitize($info['url']);
```

Comment:
`Urls::sanitize()` has been moved to `\DDtrace\Private_` because it is required by one of the functions in `\DDtrace\Private_` and from this namespace there should never be a dependency to `src/api`.

The original `Urls::sanitize()` is now a proxy to the new function.